Lib/sysconfig.py

```diff
@@ -545,15 +545,32 @@ def _init_non_posix(vars):
     """Initialize the module as appropriate for NT"""
     # set basic install directories
     import _imp
+    import re
     vars['LIBDEST'] = get_path('stdlib')
     vars['BINLIBDEST'] = get_path('platstdlib')
     vars['INCLUDEPY'] = get_path('include')
-    try:
+
+    extension_suffixes = _imp.extension_suffixes()
+    if len(extension_suffixes) >= 1:
         # GH-99201: _imp.extension_suffixes may be empty when
         # HAVE_DYNAMIC_LOADING is not set. In this case, don't set EXT_SUFFIX.
-        vars['EXT_SUFFIX'] = _imp.extension_suffixes()[0]
-    except IndexError:
-        pass
+        # e.g., "_d.cp313-win_amd64.pyd"
+        vars['EXT_SUFFIX'] = extension_suffixes[0]
+
+        # e.g., "cp313-win_amd64"
+        _, soabi, _ = vars['EXT_SUFFIX'].split('.')
+        vars['SOABI'] = soabi
+
+        # e.g., check for a "t" (for "threading") in SOABI
+        if re.match(r'cp\d+t', soabi):
+            vars['Py_NOGIL'] = 1
+
+    vars['LIBPL'] = _safe_realpath(os.path.join(get_config_var('installed_base'), 'libs'))
```

Comment:
Does the `libs` dirs on Windows contain all the stuff the POSIX one does?

```
$ ls /usr/lib/python3.11/config-3.11-x86_64-linux-gnu
config.c  config.c.in  install-sh*  libpython3.11.a  Makefile  makesetup*  __pycache__/  python-config.py*  python.o  Setup  Setup.bootstrap  Setup.local  Setup.stdlib
```

Per `LIBPL`'s description

https://github.com/python/cpython/blob/adf0f15a06c6e8ddd1a6d59b28efcbb26289f080/Makefile.pre.in#L2399-L2401

If not, and it's simply the directory where the libraries are installed, I think we should call it `LIBDIR` instead. We are trying to mimic the POSIX build vars here, so even though there might be some core differences, we should try to match things as closely as possible.

Reply:
`libs` is the directory that has to be available to the linker when compiling extension modules. Nothing else really ought to be installed there besides what CPython includes.

Whichever is the closest equivalent of that.

Reply:
I've changed the variable to `LIBDIR`